bundle/manifests/opentelemetry-operator.clusterserviceversion.yaml

```diff
@@ -414,7 +414,7 @@ spec:
   maturity: alpha
   provider:
     name: OpenTelemetry Community
-  version: 0.75.0
+  version: 0.76.1
```

Comment:
i think we want to keep the operator version 1:1 with the collector wherever possible, not sure that's documented rn but that's the aim iirc.

Reply:
ah yes [here's the link](https://github.com/open-telemetry/opentelemetry-operator#opentelemetry-operator-vs-opentelemetry-collector) By default, the OpenTelemetry Operator ensures consistent versioning between itself and the managed OpenTelemetryCollector resources. That is, if the OpenTelemetry Operator is based on version 0.40.0, it will create resources with an underlying OpenTelemetry Collector at version 0.40.0.